solidity/random-beacon/contracts/RandomBeaconParametersV2.sol

```diff
@@ -0,0 +1,112 @@
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract RandomBeaconParametersV2 is Ownable {
```

Comment:
This contract could be a good replacement for `RandomBeaconGovernance` introduced in #2613 to reduce the boilerplate code. I see two problems though: 
- we have two governance delays; some parameters are secured with 24h and some others with 2 weeks governance delay,
- some parameters are validated at the moment the change is requested.

Is it something we can have in this version without introducing significant complexity? I take the delay is doable but validation could be problematic 🤔

Reply:
Actually, if we were doing no validation and used the same delay for all parameters, we'd be able to eliminate `RandomBeaconGovernance` altogether and use [time lock controller from OpenZeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/TimelockController.sol) but I do not think we have this comfort, given [this](https://github.com/keep-network/keep-core/pull/2613#issuecomment-927813679) conversation.